OJP/OJP_Trips.xsd

```diff
@@ -106,18 +107,54 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
-	<xs:group name="TripPolicyGroup">
+	<xs:complexType name="AccessModesListOfEnumerations">
 		<xs:annotation>
-			<xs:documentation>Policies that control the trip search behaviour.</xs:documentation>
+			<xs:documentation>List of Access Modes.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:group ref="BaseTripPolicyGroup"/>
-			<xs:element name="ItModesToCover" type="IndividualModesEnumeration" minOccurs="0" maxOccurs="unbounded">
+			<xs:element name="AccessMode" type="ifopt:AccessModesEnumeration" minOccurs="0" maxOccurs="unbounded">
 				<xs:annotation>
-					<xs:documentation>For each mode in this list a separate monomodal trip shall be found - in addition to inter-modal solutions.</xs:documentation>
+					<xs:documentation>Access mode that should be considered.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
+	</xs:complexType>
+	<xs:group name="TripPolicyGroup">
+		<xs:annotation>
+			<xs:documentation>Policies that control the trip search behaviour.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:group ref="BaseTripPolicyGroup"/>
+			<xs:choice>
+				<xs:element name="MotorisedMainTravelMode" type="xs:boolean" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>The MotorisedMainTravelMode defines whether the trip planner works in PT mode (or pure IT mode) or includes carTransportRail and the like. If set to true (e.g. because one travels by car, truck or motorcycle), then carTransportRail, ferry are used together with roads.</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:sequence>
+					<xs:element name="ItModesToCover" type="IndividualModesEnumeration" minOccurs="0" maxOccurs="unbounded">
+						<xs:annotation>
+							<xs:documentation>For each mode in this list a separate monomodal trip shall be found - in addition to inter-modal solutions.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="IncludeAlternativeOptions" type="xs:boolean" default="false" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Whether alternative options should be presented as well. Mainly important for dominated journeys or in the case of ContinuousLegs the second-best route. Should be optimised for the user expectance (see. e.g https://theses.hal.science/tel-01848737). However, what the alternative options are may vary widely depending on the optimisation methods and filters.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="IncludeRestrictedLines" type="xs:boolean" minOccurs="0" default="false">
```

Comment:
Might `IncludeAllRestrictedLines` be a better name?

Reply:
changed